shared/src/models/map-image.ts

```diff
@@ -15,12 +15,16 @@ export class MapImage {
     @Type(() => ImageProperties)
     public readonly image: ImageProperties;
 
+    @IsBoolean()
+    public readonly locked: boolean;
```

Comment:
Currently, it is possible to change the position of the `mapImage`, because only the frontend enforces it.
Either 
1. enforce it in the `MoveMapImage` reducer by throwing a `ReducerError` or
2. explain this via a comment.

I would prefer 2. so e.g.:

```
/**
 * Whether the UI should prevent position changes of the mapImage.
 */
```